medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -128,12 +113,11 @@ def _object_to_dict(self, tvdb_response, key_mapping=None, list_separator='|'):
                         else:
                             return_dict[attribute] = value
 
-                    except Exception as e:
-                        logger.warning('Exception trying to parse attribute: %s, with exception: %s', attribute,
-                                       e.message)
+                    except Exception as error:
+                        log.warning('Exception trying to parse attribute: {0}, with exception: {1!r}', attribute, error)
```

Comment:
We changed this recently to e.message. So this should be changed to error.message.
I know your using __repr__ here. But that will not give the same result as error.message.